ext/nnstreamer/tensor_filter/tensor_filter_lua.cc

```diff
@@ -434,11 +434,13 @@ lua_subplugin::configure_instance (const GstTensorFilterProperties *prop)
 
   if (!g_file_test (prop->model_files[0], G_FILE_TEST_EXISTS)) {
     nns_logi ("Given model file does not exist. Do script mode.");
-    std::string script = g_strjoinv (",", (gchar **) prop->model_files);
-    if (luaL_dostring (L, script.c_str ()) != 0) {
+    gchar *script = g_strjoinv (",", (gchar **) prop->model_files);
```

Comment:
I understand your intention, but how about using a smart pointer here?

Reply:
Thanks for suggestion! I used the unique_ptr. PTAL